internal/beater/config/config_test.go

```diff
@@ -249,7 +254,18 @@ func TestUnpackConfig(t *testing.T) {
 					Enabled:      true,
 					ClientConfig: defaultDecodedKibanaClientConfig,
 				},
-				KibanaAgentConfig: KibanaAgentConfig{Cache: Cache{Expiration: 2 * time.Minute}},
+				DynamicAgentConfig: DynamicAgentConfig{
+					ESConfig: &elasticsearch.Config{
+						Hosts:            elasticsearch.Hosts{"localhost:9201", "localhost:9202"},
+						Protocol:         "https",
+						Timeout:          5 * time.Second,
+						APIKey:           "id:api_key",
+						MaxRetries:       3,
+						CompressionLevel: 5,
```

Comment:
Tests will fail if we remove this line because we are filling in default ES config values. If we delete this line we will expect CompressionLevel: 0 but in reality the there's still a default CompressionLevel: 5